xpallets/assets-registrar/src/lib.rs

```diff
@@ -10,196 +10,88 @@
 
 #[cfg(any(feature = "runtime-benchmarks", test))]
 mod benchmarking;
+mod default_weights;
```

Comment:
What's the reason for this change?

Reply:
I just want to keep consistent with the naming of substrate frame(pallet)

Reply:
Substrate is intended to be a general framework/library, but ChainX is a project with specific features based on Substrate more like Polkadot. In my opinion, `default_weights` in Substrate is to let the pallet users don't need to run the benchmarks on their own and can use the default benchmarking results directly, we don't need to follow this way.

Reply:
Furthermore, splitting the trait defintion and default implementation is not good for the future auto benchmarking script.